coffees_suggester.js

```diff
@@ -10,149 +10,111 @@ import {
 } from "./coffee_classification.js";
 import dedent from "dedent";
 
-function suggestCoffees() {
+async function suggestCoffees() {
   console.log(dedent`
     カルディコーヒーファームで取り扱っている23種類のラインナップからあなたにぴったりのコーヒーを提案します;)
     まずはあなたのお好みを聞かせてください！\n`);
 
-  displaySuggestion();
+  const preference = await askForPreference();
+  const suggestedCoffees = await decideSuggestions(preference);
+  displaySuggestion(suggestedCoffees);
 }
 
-async function displaySuggestion() {
-  try {
-    const suggestedCoffees = await decideSuggestions();
+async function askForPreference() {
+  const preference = {};
+  const { prompt } = enquirer;
+  prompt.on("cancel", () => {
```

Comment:
この分割代入は、この関数の処理とは直接関係がないので、以下のようにしてしまってもよいかなーと思います。もしくは、import するときに分割代入してしまうのもよいかもしれません。

```suggestion
  enquirer.prompt.on("cancel", () => {
```

Reply:
ご教示いただきありがとうございます！
ご提案いただいたうち、import するときに分割代入する方法に修正いたします🙇‍♀️！

*<理由>*
`const { prompt } = enquirer;`を使用しないかたちに修正してみたところ、
`ctrl + c`でキャンセルした際に`UnhandledPromiseRejection`のエラーが出るようになり`try...catch`で記述する必要が出てしまったため。